SPEC/FILES.md

```diff
@@ -618,6 +618,7 @@ Where:
 - `options` is an optional Object that might contain the following keys:
   - `hash` is a Boolean value to return only the hash.
   - `size` is a Boolean value to return only the size.
+  - `with-local` is a Boolean value to compute the amount of the dag that is local, and if possible the total size.
```

Comment:
When options have `-`, we translate them to camel case. `withLocal`

Reply:
Will do, but it's currently not respected in js-ipfs-api. For instance: https://github.com/ipfs/js-ipfs-api/blob/master/test/files.spec.js#L93

Reply:
Seems that one passed by review. That said, we still want to keep the camelCase version and support the dash for backwards compatibility

Reply:
I see that lodash is used exactly once in js-ipfs-api. Nothing against using `opts = _.mapKeys(opts, (v, k) => _.kebabCase(k))` ?

That would support both version and some more easily.